todoapp/build.gradle

```diff
@@ -29,13 +29,13 @@ ext {
     buildToolsVersion = '23.0.2'
 
     // App dependencies
-    supportLibraryVersion = '23.4.0'
+    supportLibraryVersion = '24.1.1'
```

Comment:
if we are doing this here, let's make sure a new PR is open once this is merged when we update all the libraries